### ISSUE TITLE
parglare win parse error

### ISSUE THREAD

**Issue by aleksandarbos** [igordejanovic/parglare#93]
* parglare version: 0.8.0
* Python version: 3.6.4
* Operating System: Win 7

### Description

I was trying to parse an ordinance docx file when the error occurred.

### What I Did

```
(oll) λ oll utils parse "2017 Ord 5 Accessory & Junior Dwelling Units REVISED.docx"
Parsing oll.partners.us.ca.cities.san_mateo:code with version latest
Traceback (most recent call last):
  File "C:\Users\twentyeight\Envs\oll\Scripts\oll-script.py", line 11, in <module>
    load_entry_point('oll-core', 'console_scripts', 'oll')()
  File "c:\users\twentyeight\envs\oll\lib\site-packages\pkg_resources\__init__.py", line 479, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "c:\users\twentyeight\envs\oll\lib\site-packages\pkg_resources\__init__.py", line 2703, in load_entry_point
    return ep.load()
  File "c:\users\twentyeight\envs\oll\lib\site-packages\pkg_resources\__init__.py", line 2321, in load
    return self.resolve()
  File "c:\users\twentyeight\envs\oll\lib\site-packages\pkg_resources\__init__.py", line 2327, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "e:\openlaw\platform\core\oll\tools\cli.py", line 34, in <module>
    oll()
  File "c:\users\twentyeight\envs\oll\lib\site-packages\click\core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "c:\users\twentyeight\envs\oll\lib\site-packages\click\core.py", line 697, in main
    rv = self.invoke(ctx)
  File "c:\users\twentyeight\envs\oll\lib\site-packages\click\core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\users\twentyeight\envs\oll\lib\site-packages\click\core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\users\twentyeight\envs\oll\lib\site-packages\click\core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\users\twentyeight\envs\oll\lib\site-packages\click\core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "e:\openlaw\platform\core\oll\tools\utils\__init__.py", line 94, in parse
    dump_parsed=dump_parsed)
  File "e:\openlaw\platform\core\oll\parser\__init__.py", line 62, in get_parser
    return document_element.draft_parser_latest
  File "e:\openlaw\platform\core\oll\core\utils.py", line 161, in __get__
    return self.f(owner)
  File "e:\openlaw\platform\core\oll\core\models\DocumentEl\__init__.py", line 184, in draft_parser_latest
    return cls.get_draft_parsers()[cls._draft_parser_latest_version]
  File "e:\openlaw\platform\core\oll\core\utils.py", line 195, in __getitem__
    value = value(**self.kwargs)
  File "e:\openlaw\platform\core\oll\parser\__init__.py", line 194, in __init__
    self._construct_parser()
  File "e:\openlaw\platform\core\oll\parser\__init__.py", line 351, in _construct_parser
    debug=self.debug, debug_colors=self.debug)
  File "c:\users\twentyeight\envs\oll\lib\site-packages\parglare\glr.py", line 64, in __init__
    force_load_table=force_load_table)
  File "c:\users\twentyeight\envs\oll\lib\site-packages\parglare\parser.py", line 86, in __init__
    in_layout=self.in_layout)
  File "c:\users\twentyeight\envs\oll\lib\site-packages\parglare\tables\__init__.py", line 90, in create_load_table
    table = load_table(table_file_name, grammar)
  File "c:\users\twentyeight\envs\oll\lib\site-packages\parglare\tables\persist.py", line 46, in load_table
    act_prod = grammar.productions[json_action['prod_id']]
IndexError: list index out of range
```

**Comment by igordejanovic** [comment]
@aleksandarbos I think that this issue was due to bug fixed in 0.9.1
If you don't have any more problems with this please close the issue.

**Comment by aleksandarbos** [comment]
The issue no longer occurs.
